### PR TITLE
Adds basic documentation

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

**Comment by GaretJax** [comment]
Don't commit the `_build` dir (and add it to .gitignore)

**Comment by mkoistinen** [comment]
Thanks, @GaretJax, sorted!

File: docs/development.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Aldryn Categories uses `django CMS Helper <https://github.com/nephila/djangocms-helper>`_ to run its
+test suite.
+
+Backend Tests
+=============
+
+To run the tests, in the aldryn-categories directory::
+
+    virtualenv env  # create a virtual environment
```
Comment (FinalAngel) [inline docs/development.rst]: does the test setup here work, did you try?

File: docs/development.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+Frontend Tests
+==============
+
+Follow the instructions in the `aldryn-boilerplate-bootstrap3
+<https://aldryn-boilerplate-bootstrap3.readthedocs.org/en/latest/testing/index.html>`_
+documentation and setup the environment through the `Backend Tests` section.
+
+Instead of using ``python test_settings.py`` described above, you need to
```
Comment (FinalAngel) [inline docs/development.rst]: no frontend tests here

**Comment by stefanfoulis** [comment]
:+1:

File: docs/development.rst (hunk 2 above)
Comment (mkoistinen) [inline docs/development.rst]: Yes, this is true.

File: docs/development.rst (hunk 1 above)
Comment (mkoistinen) [inline docs/development.rst]: Yep.

**Pull request merged**